src/server/controllers/verifyEmail.js

```diff
@@ -0,0 +1,9 @@
+const { updateEmailVerification } = require('./../../database/db_update');
+
+module.exports = (req, res) => {
+
+ updateEmailVerification(req.params.username, req.params.token, (error, response) => {
+   if (error) { res.status(500).send(error) };
```

Comment:
okay, there are two styles you can use
```js
if (err) {
  res.send(err)
  return
}
res.send(data)

// shorthand
if (err) return res.send(err)
res.send(data)
```

OR

```js
if (err) {
  res.send(err)
} else {
  res.send(data)
}
```

See that both these examples will only try to send once, unlike yours, which will try send twice on error